core/config.go

```diff
@@ -35,4 +77,8 @@ type Config struct {
 	// time.Sleep without actually sleeping. While testing it defaults to 0 (which won't sleep at all), in
 	// real-world usage it's expected to be set to 1
 	SleepMultiplier time.Duration
+
+	// Filter on extra ASG tags in addition to finding those
+	// ASGs with spot-enabled=true
+	FilterByTags Tags
```

Comment:
I guess the code could be tested despite its simplicity, just for the sake of it

Reply:
Are there any tests for the Config at the moment, I'm not finding any, but perhaps I'm not looking in the correct place.

How do we currently test the Config object?